environment.yml

```diff
@@ -23,16 +23,17 @@ dependencies:
   - tqdm
   - pyfftw<=0.12.0  # used by ghostipy. install from conda-forge so that it works on Mac ARM processors
   - pip:
-    - pubnub<6.4.0
-    - git+https://github.com/SpikeInterface/spikeinterface.git
-    - pynwb>=2.2.0,<3
-    - hdmf>=3.4.6
-    - datajoint>=0.13.6
-    - ghostipy
-    - pymysql
-    - sortingview>=0.11
-    - figurl-jupyter
-    - git+https://github.com/LorenFrankLab/ndx-franklab-novela.git
-    - pyyaml
-    - click
-    - 'black[jupyter]'
+      - pubnub<6.4.0
+      - git+https://github.com/SpikeInterface/spikeinterface.git
+      - pynwb>=2.2.0,<3
+      - hdmf>=3.4.6
+      - datajoint>=0.13.6
+      - ghostipy
+      - pymysql
+      - sortingview>=0.11
+      - figurl-jupyter
+      - git+https://github.com/LorenFrankLab/ndx-franklab-novela.git
+      - pyyaml
+      - click
+      - "black[jupyter]"
+      - .
```

Comment:
Out of curiosity, what does the `.` do in this context? If it acts like `pip install .` does it install an editable version? Also curious how it knows to install spyglass from that line? Does it assume that the environment.yml is in the same directory as the cloned spyglass repo?

Reply:
On my other projects, that's how `.` or `-e .` worked yes, editable in the latter case. Yes, it assumes you left the env yaml in the root repo dir when cloned

Reply:
So if someone were to want to install an editable version into their environment would they have to change this line or risk having conflicting installs of spyglass? I get that this is most likely done with a wider release in mind, but just wanted to clarify the effects.